.github/workflows/filter_testsuite.yml

```diff
@@ -0,0 +1,105 @@
+name: Filter Testsuite
+
+on:
+  issue_comment:
+      types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v3
+        id: check
+        with:
+         result-encoding: string
+         script: |
+           const userName = context.payload.comment.user.login
+           if(userName == 'maxGimeno') {
+             const body = context.payload.comment.body
+             if(body.includes("/testme")) {             
+               return 'OK'
+             }
+           }
+           return 'stop'
```

Comment:
This step should be replaced by a [`jobs.<job>.if`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif), because `comment.user.login` and `comment.body` are directly usable in a Github Action expression, and there exists the function [`contains`](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#contains).

I will probably merge the PR anyway, to test it on https://github.com/CGAL/cgal, but my suggestion will speed-up the reaction time of our Github Actions: a `step` forks a container on a runner, and that is "long" (waiting in the wait queue, then about 15 seconds), whereas expressions seem to be evaluated even before entering the way queue (there is probably another wait queue that we do not see).